content/en/serverless/azure_app_services.md

```diff
@@ -135,6 +135,10 @@ Console.WriteLine(ex);
 DogStatsd.Increment("sample.startup");
 ```
 
+<div class="alert alert-info">Note that if you want to send custom metrics while disabling tracing altogether, you can set the following variables in your application's config:
```

Comment:
```suggestion
<div class="alert alert-info">**Note**: To send only custom metrics (while disabling tracing) set the following variables in your application's config:
```

Reply:
Fixed in 0d7b9c5ffb638618c789d280c59c44b4aed688c6 (as it's HTML, the markdown notation for bold won't work)

Reply:
>as it's HTML, the markdown notation for bold won't work

In which case, will 

```
Set `DD_TRACE_ENABLED` to `false`
```

work? 🤔